Apps/W1/EDocumentConnectors/Tietoevry/app/src/Permissions/TietoevryEDocConnectorEdit.PermissionSetExt.al

```diff
@@ -0,0 +1,12 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Tietoevry;
+
+using System.Security.AccessControl;
+
+permissionsetextension 6394 "Tietoevry EDoc. Connector - Edit" extends "D365 BASIC"
```

Comment:
Extending d365 basic. Check that it is ok.

Reply:
?